TemplateProject/fastlane/Fastfile

```diff
@@ -0,0 +1,71 @@
+desc "Bump the iOS and Android version with a commit and tag."
+lane :bump_version do |options|
+  type = options.fetch(:type)
+
+  case type
+  when "major", "minor", "patch"
```

Comment:
Oh wait, I guess if it gets to the commit stage it will technically bump the build since the commits increased. Even with that I'm still thinking raising is correct since we don't tag or commit other build bumps, it could be odd that we do with this.

Perhaps we reevaluate if we find we're needing to manually bump the build num often enough that providing a command for that is useful?